Kudu.Core.Test/Infrastructure/XmlUtilityFacts.cs

```diff
@@ -20,15 +19,17 @@ public void XmlUtilityReturnsValidCharacterStringsUnchanged(string input)
             Assert.Equal(input, output);
         }
 
-        [Theory]
-        [PropertyData("InvalidCharacterSet")]
-        public void XmlUtilityRemovesInvalidCharacters(string input)
+        [Fact]
```

Comment:
this is a regression from xunit 1.9 when saving the result.  It tries to put that input value to XML result and choke due to invalid Xml character.

https://github.com/xunit/xunit/issues/380